packages/protocol/contracts/identity/Attestations.sol

```diff
@@ -238,46 +230,6 @@ contract Attestations is
     delete state.unselectedRequests[msg.sender];
   }
 
-  /**
```

Comment:
Document this in the PR desrcriptinn. If anything, maybe this should be the title of the PR